Package.swift

```diff
@@ -16,7 +16,9 @@ let package = Package(
         .library(name: "NotarizeCV",
                          targets: ["NotarizeCV"]),
         .library(name: "opencv2",
-                         targets: ["opencv2"])
+                         targets: ["opencv2"]),
+        .library(name: "PSPDFKit",
```

Comment:
I think we need to make this a dependency on NotarizeDeps instead of being a library available via this repo. I.e in line 59 we add the binary targets we defined as dependencies of the NotarizeDeps target